docs/guide.md

```diff
@@ -0,0 +1,103 @@
+# User Guide
+
+## Session
+
+Communication with the Planet services is provided with the `Session` class.
+The recommended way to use a `Session` is as a context manager. This will
+provide for automatic clean up of connections when the context is left.
+
+```python
+>>> import asyncio
+>>> import os
+>>> from planet import Session
+>>>
+>>> AUTH = (os.getenv('PL_API_KEY'), '')
+>>>
+>>> async def main():
+...     async with Session(auth=AUTH) as sess:
+...         # perform operations here
+...         pass
+...
+>>> asyncio.run(main())
+
+```
+
+Alternatively, use `await Session.aclose()` to close a `Session` explicitly:
+
+```python
+>>> async def main():
+...     sess = Session(auth=AUTH)
+...     # perform operations here
+...     await sess.aclose()
+...
+>>> asyncio.run(main())
+
+```
+
+## Orders Client
+
+The Orders Client mostly mirrors the
+[Orders API](https://developers.planet.com/docs/orders/reference/#tag/Orders),
+with the only difference being the addition of the ability to poll for when an
+order is completed and to download an entire order.
+
+```python
+>>> from planet import OrdersClient
+>>>
+>>> async def main():
+...     async with Session(auth=AUTH) as sess:
+...         client = OrdersClient(sess)
+...         # perform operations here
+...
+>>> asyncio.run(main())
+
+```
+
+### Creating an Order
+
+When creating an order, the order details must be provided to the API. There
+are two ways to specify the order details, a `JSON` blob and an `OrderDetails`
+instance.
+
+An `OrderDetails` instance is built up from instances of `Product`, `Tool`, 
+`Delivery`, and `Notification` alongside other, simple parameters.
+
+An example of creating an `OrderDetails` instance:
+
+```python
+>>> from planet.api.order_details import OrderDetails, Product
+>>>
+>>> image_ids = ['3949357_1454705_2020-12-01_241c']
+>>> order_detail = OrderDetails(
+...     'test_order',
+...     [Product(image_ids, 'analytic', 'psorthotile')]
+... )
+...
+
+```
+
+The same thing, expressed as a `JSON` blob (pro tip: this can be obtained
+with `order_detail.json`):
+
+```python
+>>> order_detail = {
+...     'name': 'test_order',
+...     'products': [{'item_ids': ['3949357_1454705_2020-12-01_241c'],
+...                   'item_type': 'PSOrthoTile',
+...                   'product_bundle': 'analytic'}],
+... }
+
+```
+
+Once the order details are built up, creating an order is done within
+the context of a `Session` with the `OrdersClient`:
+
+```python
+>>> async def main():
+...     async with Session(auth=AUTH) as sess:
+...         cl = OrdersClient(sess)
+...         order_id = await cl.create_order(order_detail)
+...
+>>> asyncio.run(main())
```

Comment:
just to echo what @strixcuriosus said, let's make codeblocks copy-pastable and remove the prefix / line marker things

Reply:
I am very with you on this. I will look into if it is possible to doctest without the prefix. At first look, it seemed they are required for doctest to identify the code. Maybe there is an alternative.

Reply:
Lets continue this on #275